.changelog/326.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:new-datasource
+`datasource/cloudavenue_storage_profile` is a new datasource that can be used to retrieve the details od storage profile.
```

Comment:
syntax : is a new datasource that can be used to retrieve the details of storage profile.